src/UI/Component/Button/Factory.php

```diff
@@ -112,4 +112,29 @@ public function primary($label, $action);
 	 * @return  \ILIAS\UI\Component\Button\Close
 	 */
 	public function close();
+
+	/**
+	 * ---
+	 * description:
+	 *   purpose: >
+	 *       The Split Button is used to group related actions if space is scarce while offering the most prominent as
+	 *       visible default.
+	 *   composition: >
+	 *       The Split Button is composed of a Standard Button offering the default action along with a dropdown containing all further options.
```

Comment:
Thanks for the hint with the characters, changed my settings in PHP-Storms accordingly.